fix(backend port number): change backend port number to 6174

zxt-tzx opened the pull request:
Current port number of 5000 clashes with Apple using port 5000 for AirPlay Receiver on Mac.

fixes #992

Notes:
- Port number of 6174 is somewhat arbitrarily chosen. 
- Not sure if it's possible to make the port number a variable instead (so as to adhere to DRY principle); currently, the port number is repeated across multiple files)

Pros: macOS users don't need to switch off AirPlay. (Useful when, say, trying to use Mac to stream audio/video from iPhone!) 
Cons: Defies convention of using 5000 for ExpressJS. 6174 chosen [somewhat arbitrarily](https://en.wikipedia.org/wiki/6174_(number)).  It also appear [to be unassigned](https://www.adminsub.net/tcp-udp-port-finder/6174).

LoneRifle reviewed (changes requested):
Lgtm otherwise

LoneRifle commented on server/src/bootstrap/index.ts:
```diff
@@ -77,7 +77,7 @@ app.use(requestLoggingMiddleware)
 // cors enable
 app.options('*', cors<express.Request>())
-app.use(cors({ origin: 'http://localhost:5000' }))
+app.use(cors({ origin: 'http://localhost:6174' }))
```
Have this replaced by `SERVER_PORT` instead

zxt-tzx replied:
OK done!

zxt-tzx committed:
fix(backend port number): amend local backend port number to 6174

Amend local backend port number to 6174 to avoid conflict with AirPlay Receiver, which is set to
5000 starting in macOS 12.

fix #992

LoneRifle reviewed (approved):
lgtm
